icu4c/source/data/brkitr/root.txt

```diff
@@ -22,8 +22,4 @@ root{
         Mymr:process(dependency){"burmesedict.dict"}
         Thai:process(dependency){"thaidict.dict"}
     }
-    lstm{
-        Thai{"Thai_graphclust_model4_heavy.res"}
-        Mymr{"Burmese_graphclust_model5_heavy.res"}
-    }
```

Comment:
Why did you delete these lines? I think you should not delete them.

Reply:
See the email thread “Have you ever converted the lstm data to ICU4J”

Reply:
I think Shane was not on that thread. These lines are not supposed to be added manually to data/brkitr/root.txt; they are only supposed be added via additions to data/xml/brkitr/root.xml (and if not there, the integration process will delete the lines in root.txt). In order to start the process of adding the support in data/xml/brkitr/root.xml, I had to make some dtd additions and changes to tool/cldrtoicu/mapper/BreakIteratorMapper.java. But then we decided to not automatically include the lstm data; consequently I commented out the relevant additions in data/xml/brkitr/root.xml (if I do not do that, the creation of ICU4J jars fails with an unresolved dependency).

Reply:
Thanks Markus for forwarding the thread.

***It seems like removing these lines breaks the LSTM.***  Frank added these lines to root.txt as an optional dependency.  If the tooling doesn't like these lines, then we should open a ticket to fix the tooling, and consider these lines as a temporary patch until the tooling is fixed.  Evidence: on mainline, the TestLSTMThai test takes 180 ms to run ([source](https://github.com/unicode-org/icu/runs/3366503463)):

```
2021-08-18T23:30:15.1191864Z          TestLSTMThai {
2021-08-18T23:30:15.2987089Z          } OK:   TestLSTMThai  (180ms) 
```

However, on this PR, the test takes less than 1 ms, suggesting that the test gets skipped ([source](https://github.com/unicode-org/icu/pull/1802/checks?check_run_id=3366784491)):

```
2021-08-19T00:18:34.7485907Z          TestLSTMThai {
2021-08-19T00:18:34.7486498Z          } OK:   TestLSTMThai 
```

Here is the "skip test" line taht gets run unless the data is present in the bundle:

https://github.com/unicode-org/icu/blob/c26aebe802f1a87483f70ad7557fa0ce52061eb9/icu4c/source/test/intltest/rbbitst.cpp#L5261

Reply:
I can re-add the lines but I will do that as part of the alpha1 integration, which I am starting now and should have done within a day. I want to merge this PR before I start that integration so I do not want to make those changes in this PR. I am running out of time.